test/controllers/posts_controller_test.exs

```diff
@@ -0,0 +1,73 @@
+defmodule ElixirJobBoard.PostsControllerTest do
+  use ElixirJobBoard.ConnCase
+  alias ElixirJobBoard.Job
+  import ElixirJobBoard.Factory
+  require IEx
```

Comment:
This allows to use ```IEx.pry``` like you use binding.pry in Ruby. The only other thing that you have is run ```iex -S mix test``` from the command line when you are running tests instead of ```mix test```.